include/osem.h

```diff
@@ -39,12 +39,14 @@ typedef struct {
 	uint16_t maxLateralDeviation;
 	uint16_t maxYawDeviation;
 	uint16_t maxPositionError;
-	uint16_t heabTimeout;
+	uint16_t communicationTimeout;
 	uint8_t testMode;
-	uint8_t monrRate;
-	uint8_t monr2Rate;
-	uint8_t heabRate;
+	uint16_t monrRate;
+	uint16_t monr2Rate;
 	uint32_t maxMessageLength;
+	uint8_t emergencyBehavior;
+	uint8_t comLost;
```

Comment:
```suggestion
	uint8_t communicationLost;
```
If we want to keep it consistent with `communicationTimeout`

Reply:
Using the names used in the iso22133 document, like we have done for all other parameters